wp-content/plugins/wporg-learn/inc/post-meta.php

```diff
@@ -384,6 +386,19 @@ function add_workshop_metaboxes() {
 	);
 }
 
+/**
+ * Add meta boxes to the Edit Meeting screen.
+ */
+function add_meeting_metaboxes( $post_type = '' ) {
+	add_meta_box(
+		'meeting-language',
+		__( 'Meeting Language', 'wporg_learn' ),
```

Comment:
IMO this isn't necessary, and using just 'Language' makes it consistent with other meta boxes
```suggestion
		__( 'Language', 'wporg_learn' ),
```

Reply:
tbh I had thought of that but just left it in the end, so good to have confirmation that you thought the same.